README.md

```diff
@@ -36,27 +38,27 @@ Service (PECS) programme.
     npm run build
     ```
 
-## Configuring the application
+### Configuring the application
 
 Create a copy of the example environment variable file and add values for the keys:
```

Comment:
Could we keep an example `.env` in one our private repos? Rather than doing the usual "Who's got something that works?" dance…

Reply:
The `env` file has auth details, urls, and permission overrides. Apart from the auth, the other information shouldn't need to be extra-secure.

If we've got a repo that would work, maybe we could stick it there. Or maybe in a shared document somewhere that's not a repo? Non technical users might want the username/passwords too?

Maybe @martin-ballhatchet-moj has some thoughts on this?

Reply:
For those that need access to the real environment variables and/or usernames, the process to get these should be covered in our internal documentation (e.g. Confluence) - potentially as an "ask this group of people for the right values". I think there are secret sharing applications in use at MoJ, but those might be overkill. For non-secret things, happy for there to be an example env.

Reply:
> The `env` file has auth details, urls, and permission overrides. Apart from the auth, the other information shouldn't need to be extra-secure.

Having the .env stored in our private repo that is git-encypted seems no more problematic than storing our actual production secrets. Certainly less of an issue than copying and pasting it in to Slack....